explainaboard/loaders/file_loader.py

```diff
@@ -62,6 +62,7 @@ class FileLoaderField:
     dtype: Optional[DType] = None
     strip_before_parsing: Optional[bool] = None
     parser: Optional[Callable[[str], str]] = None
+    skippable: Optional[bool] = False
```

Comment:
Ditto
```suggestion
    optional: bool = False
```

Reply:
`optional: bool = False` needs to be put before all `Optional` variables. Since some `FileLoaderField` definitions specify dtype without `dtype=`, it's better to put it at the end as `optional: Optional[bool] = None` similar to `strip_before_parsing`.